lib/transmuxer/transmuxable.rb

```diff
@@ -51,6 +57,15 @@ def transmux_retry
       Transmuxer::Job.resubmit(zencoder_job_id)
     end
 
+    def transmux_restart
+      Transmuxer::Job.cancel(zencoder_job_id)
```

Comment:
How about we move `Transmuxer::Job.cancel` and resetting the Zencoder AR columns to a separate method `transmux_cancel`. Then `transmux_restart` will be simply `transmux_cancel && transmux`